modules/nw-ingress-sharding-default.adoc

```diff
@@ -0,0 +1,55 @@
+// Module include in the following assemblies:
+//
+// * ingress-operator.adoc
+// * networking/ingress-sharding.adoc
+
+:_content-type: PROCEDURE
+[id="nw-ingress-sharding-default_{context}"]
+= Sharding the default Ingress Controller
+
+After creating a new Ingress shard, there could be routes that are admitted for your new Ingress shard that are also admitted by the default Ingress Controller. This is because the default Ingress Controller has no selectors and admits all routes by default.
+
+The following procedure restricts the default Ingress Controller from servicing your newly sharded `finance`, `ops`, and `dev`, routes. This adds further isolation to Ingress shards.
+
+[IMPORTANT]
+====
+You must keep all of {product-title}'s administration routes on the same Ingress Controller. Therefore, use the default Ingress Controller as outlined by the following procedure. Avoid adding additional selectors to the default Ingress Controller that exclude these essential routes.
+====
+
+.Prerequisites
+
+* You installed the OpenShift CLI (`oc`).
+* You are logged in as a project administrator.
+
+.Procedure
+
+. Run the following command to modify the default Ingress Controller:
++
+[source,terminal]
+----
+$ oc edit ingresscontroller -n openshift-ingress-operator default
+----
+
+. Edit the Ingress Controller to contain a `namespaceSelector` that excludes the `finance`, `ops`, and `dev` routes:
```

Comment:
Good point. I'll add this sentence to the abstract just before the procedure begins. That way, we can be general but mention the procedure will use one or the other. This will keep the procedure itself as unambiguous as possible